arrows/vtk/vtk_version.h.in

```diff
@@ -0,0 +1,17 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+/// \file
+/// \brief Macro to define VTK version
+
+/////////////////////////////////////////////////////////////////////////////
+// This file is generated by CMake. Do not edit. All changes will be lost. //
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef KWIVER_ARROWS_VTK_VERSION_H
+#define KWIVER_ARROWS_VTK_VERSION_H
+
+#cmakedefine VTK_VERSION_9_1
```

Comment:
This works, but it might be more generally useful to just use `VTK_VERSION` here, which is already defined in CMake.  Then in the `.cxx` you can use `#if VTK_VERSION >= 9.1`.  That way this will work for future versions of VTK and be more clear that it's not just VTK 9.1 but VTK >= 9.1

Reply:
`VTK_VERSION` is a string and relies on [CMake version comparisons](https://cmake.org/cmake/help/latest/command/if.html#version-comparisons) to be compared. Is there an elegant way to make this comparison in the `.cxx` side of things?

Reply:
You might be able to use `VTK_VERSION_MAJOR` and `VTK_VERSION_MINOR` instead.
```c++
#if VTK_VERSION_MAJOR >=9 && VTK_VERSION_MINOR >= 1
```

Reply:
It doesn't seem like those are defined in the scope of KWIVER. I think that the original solution is cleaner since `VTK_VERSION VERSION_GREATER_EQUAL 9.1` encompasses all of the desired functionality and the other methods don't seem to be well supported.

Reply:
Another suggestion would be to use `add_definitions` to set a preprocessor flag like I did [here](https://github.com/Kitware/kwiver/blob/472993f1d6ae3981e240951ea812a7a7f8f46d95/arrows/vxl/CMakeLists.txt#L79) for vxl and ffmpeg. This can then be used directly in the source [code](https://github.com/Kitware/kwiver/blob/472993f1d6ae3981e240951ea812a7a7f8f46d95/arrows/vxl/register_algorithms.cxx#L32-L34) with `#ifdef` without having to use a generated header file.